wideq/core.py

```diff
@@ -366,6 +368,18 @@ def monitor_stop(self, device_id, work_id):
             'workId': work_id,
         })
 
+    def set_device_operation(self, device_id, values):
+        """ some device can use this for operation"""
```

Comment:
Any chance this docstring could clarify something about what the difference is between "operation" and "controls" is?